src/Search/Facets.jsx

```diff
@@ -132,50 +101,41 @@ const Tabs = styled.div`
   }
 `;
 
-const TabsButton =
-  props.tabButtonStyle ??
-  styled("Link")`
-    display: inline-flex;
-    align-items: center;
-    justify-content: center;
-    height: 100%;
-    font-weight: 600;
-    font-size: 12px;
-    padding: 0 12px;
-    position: relative;
-    color: ${(p) => (p.selected ? "#11181C" : "#687076")};
-    background: none;
-    border: none;
-    outline: none;
-    text-align: center;
-    text-decoration: none !important;
-    flex: 1;
-
-    &:hover {
-      color: #ffffff;
-      border-bottom: 2px solid #9799f8;
-    }
-
-    &::after {
-      content: "";
-      display: ${(p) => (p.selected ? "block" : "none")};
-      position: absolute;
-      bottom: 0;
-      left: 0;
-      right: 0;
-      height: 3px;
-      background: #59e692;
-    }
-  `;
+const TabsButton = styled.button`
+  all: unset;
+  display: inline-flex;
+  align-items: center;
+  justify-content: center;
+  height: 100%;
+  font-weight: 600;
+  font-size: 12px;
+  padding: 8px 12px;
+  position: relative;
+  color: ${(p) => (p.selected ? "#11181C" : "#687076")};
```

Comment:
There is no `selected` here, right?

Reply:
Good catch! Forgot to remove that. Thanks!